spyder/plugins/findinfiles/widgets/main_widget.py

```diff
@@ -12,9 +12,10 @@
 
 # Third party imports
 from qtpy import PYSIDE2
-from qtpy.QtCore import Signal, Qt
+from qtpy.QtCore import Signal, QEvent, Qt
```

Comment:
```suggestion
from qtpy.QtCore import QEvent, Qt, Signal
```

Sort imports in alphabetical order.